backend/src/schemas/models/assistant.py

```
@@ -34,3 +39,25 @@ def slug(self) -> str:
     @field_serializer("created_at", "updated_at")
     def serialize_dt(self, dt: Optional[datetime], _):
         return dt.isoformat() if dt else None
+
+    def to_llm_request(
+        self,
+        messages: list[BaseMessage] = None,
```

Ruff (0.14.0)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)